javascript/builtins/String.json

```diff
@@ -1299,45 +1299,98 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/match",
             "spec_url": "https://tc39.es/ecma262/#sec-string.prototype.match",
             "support": {
-              "chrome": {
-                "version_added": "1"
-              },
-              "chrome_android": {
-                "version_added": "18"
-              },
+              "chrome": [
+                {
+                  "version_added": "64"
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "64",
+                  "notes": "No support for named capturing groups."
```

Comment:
Unless someone deliberately opens the notes on the MDN page, it will look like Chrome didn't support `match()` until 64. That is very misleading. @Elchi3 Do we have another option?

Reply:
Yes, we should probably split out this feature into its own sub-feature given it will have a different version number for each browser. It shouldn't be cramped into notes.

Reply:
If we do that, then "Named capturing groups" will appear as a separate row in the MDN rendering which is much better imo.

Reply:
FWIW: we already have a separate `named_capture_groups` entry which, looking at the history, I think @vinyldarkscratch added:

https://github.com/mdn/browser-compat-data/blob/c100ac01c6905d7b6fa521139efc2a462cc75fc0/javascript/builtins/RegExp.json#L947-L953

However we don't show this anywhere on the `String.prototype.match` MDN page. I'm assuming the set of browsers which are marked as compatible in `named_capture_groups` and those which support named capture groups in `String.prototype.match` is equivalent - so is there any way we can reference this feature data within `String.prototype.match`?

Reply:
We currently have no mechanism in place that would "join" data points, so we can't reference it in the same table. We could add a second table with a query to `named_capture_groups`. We sometimes do that, see e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#Browser_compatibility.
Or we duplicate the data and then it will appear in the String.prototype.match table. We sometimes do that, too.